pkg/server/api_buildimage.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	jobStatusStarted = "Started"
+	//	jobStatusCompleted  = "Completed"
```

Comment:
not sure why this is a comment, even I guess it is not relevant at this stage

Reply:
It was just to make the linter happy (as right now that const is not used 😅 )